app/lib/service/entrypoint/dartdoc.dart

```diff
@@ -53,6 +53,7 @@ class DartdocCommand extends Command {
       frontendEntryPoint: _frontendMain,
       workerSetup: workerSetup,
       workerEntryPoint: _workerMain,
+      deadWorkerTimeout: Duration(hours: 4),
```

Comment:
so if a worker doesn't ping us back in 4-8 hours it gets killed? Seems like 1 hour would be enough? Should any task ever take 1 hour?

Reply:
or maybe 2 or 4 :D hehe, just thinking...

Reply:
1 hour works too. I wanted to be cautious, but even 1 hour is much more than our standard limit (e.g. I think we kill processes after 10 minutes or so).